src/Core/ViewHelper/TagBuilder.php

```diff
@@ -190,9 +193,14 @@ public function addAttribute($attributeName, $attributeValue, $escapeSpecialChar
         if ($escapeSpecialCharacters) {
             $attributeName = htmlspecialchars($attributeName);
         }
-        if (in_array($attributeName, ['data', 'aria'], true)
-            && (is_array($attributeValue) || $attributeValue instanceof \Traversable)
-        ) {
+        if (is_array($attributeValue) || $attributeValue instanceof \Traversable) {
+            if (!in_array($attributeName, ['data', 'aria'], true)) {
+                throw new \InvalidArgumentException(
+                    sprintf('Specified tag attribute "%s" cannot be of type array.', $attributeName),
```

Comment:
wording suggestion: "Value of tag attribute %s can't be of type array"

Reply:
fixed